POPTLC.Settings.xml

```diff
@@ -33,28 +49,26 @@
         <!-- <Setting Id="boss__7414__UPP_04" Label="Vahram 1" State="false" /> -->
         <Setting Id="boss__7413__CIT_DEP_09_CorruptedSargon" Label="Alternate Sargon (The Depths)" State="false" />
         <Setting Id="boss__2927__HYR_SOM_14_ForestQueen" Label="Kiana" State="false" />
-        <Setting Id="boss__7417__END_PIT_20_AzdahaArena" Label="Azdaha" State="false" />
+        <Setting Id="boss__7417__END_PIT_20_AzdahaArena" Label="Azhdaha" State="false" />
         <Setting Id="boss__7419__VAL_RAG_13_Orod" Label="Orod" State="false" />
         <Setting Id="boss__7420__CIT_UPP_06_Menolias_Arena" Label="Menolias" State="false" />
         <Setting Id="boss__7414__CIT_UPP_04_Varham_Arena" Label="Vahram 2" State="false" />
         <Setting Id="boss__7418__NEC_TOW_04_DARIUS" Label="King Darius" State="false" />
-        <!-- Custom setting for the end of the game -->
-        <Setting Id="vahram3" Label="Vahram 3 (END SPLIT)" State="true" />
         <Setting Id="boss__3454__CIT_DEP_11_UndeadErlik" Label="Undead Erlik" State="false" />
         <Setting Id="boss__7548__CIT_DEP02_03_nolight2_sphincterhall" Label="Giant Crab" State="false" />
         <Setting Id="boss__7056__CIT_LOW_22_Radjen" Label="Ardashir" State="false" />
         <!-- <Setting Id="boss__7413__270" Label="Alternate Sargon (Sacred Archives)" State="false" /> -->
         <Setting Id="boss__7413__HYR_SOM_22_CorruptedTwinSargon" Label="Alternate Sargons (Hyrcanian Forest)" State="false" />
         <Setting Id="boss__7413__END_PIT_15_CorruptedSargon" Label="Alternate Sargon (Pit of Eternal Sand)" State="false" />
     </Setting>
-
+    
     <Setting Id="quest" Label="Split on completing a quest" State="true">
-        <Setting Id="inlevel_OLD_01" Label="Persia Under Attack (Battlefield + Thomyris Palace)" State="false" />
+        <Setting Id="quest_end_e36c64cd5a982ce4b8571f8190c1ad43" Label="Persia Under Attack" State="false" ToolTip="Splits on finishing Thomyris Palace (Unreliable)" />
```

Comment:
FYI this is fundamentally broken, we should not include this quest split

This is why those level splits exist, they make these quest splits reliable

Reply:
The same goes for the Vahram 3 QTE - it's to make the vahram 3 boss kill reliable.

And the same could go for menolias & orod QTE if we prefer it to split then. But we should pick just one.

Reply:
Removed `quest_end_e36c64cd5a982ce4b8571f8190c1ad43` from settings.